pydp/__init__.py

```diff
@@ -1 +1,3 @@
 from .pydp import *
+
+__version__ = "0.1.1"
```

Comment:
can this upgrade version when we use bumpversion package?

Reply:
no, this is the most minimalist method I found. Others require to have a different version file or read from a txt file.
I don't know about bumpversion, but is it possible to add this section so 2 parts are updated when bumpversion is called?

Reply:
going to investigate bumpversion and see I could make it edit 2 files when called.

Reply:
This is just a thought. If this can't be fixed, it's alright.